pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -293,6 +293,76 @@ func Test_EvenDistributionOfReplicas(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		// Test Case No.5 of even distribution of replicas
+		// 1. create deployment (replicas=6), weight=1:1:1:1
+		// 2. check four member cluster replicas, can be 2:2:1:1、2:1:2:1、2:1:1:2、1:2:2:1、1:2:1:2、1:1:2:2
+		// 3. change static weight from 1:1:1:1 to 2:1:1:1
+		// 4. check four member cluster replicas, the result should be 3:1:1:1
```

Comment:
I suggest clearly stating the objectives of the current test case in the comments, such as testing for scaling, downsizing, or weight changes. Remove specific data and refer to it in the test content instead. This will help viewers of the test quickly grasp its purpose.

Reply:
ok, updated~